content/docs/command-reference/exp/branch.md

```diff
@@ -0,0 +1,38 @@
+# exp branch
+
+Commit the results from any [experiment](/doc/command-reference/exp) to Git, in
+a new branch (which will become the current <abbr>workspace</abbr>).
```

Comment:
Another Q on `exp branch`:

> which will become the current workspace

Is this right? `git checkout <branch>` happens?
And what happens if there's unstaged changes in the working tree, does the command fail with a `git` error message? Are they lost? Thanks

Reply:
`exp branch` does not checkout the branch. It just creates it, the same way that `git branch foo` just creates a branch named foo but does not checkout the new branch.

Reply:
Ooooh OK thanks, I had that wrong. I tried it and noticed that so this is fixed now ✔️